runtime/imported_values_memory_metering_test.go

```diff
@@ -46,12 +46,13 @@ func TestImportedValueMemoryMetering(t *testing.T) {
 	runtime := newTestInterpreterRuntime()
 
 	runtimeInterface := func(meter map[common.MemoryKind]uint64) *testRuntimeInterface {
-		return &testRuntimeInterface{
+		intf := &testRuntimeInterface{
 			meterMemory: testUseMemory(meter),
-			decodeArgument: func(b []byte, t cadence.Type) (cadence.Value, error) {
-				return jsoncdc.Decode(b)
-			},
 		}
+		intf.decodeArgument = func(b []byte, t cadence.Type) (cadence.Value, error) {
```

Comment:
Maybe add some test for metering the locations passed as part arguments (e.g: the ones created via the JSON decoder)